man/tpm2_policycommandcode.1.md

```diff
@@ -0,0 +1,69 @@
+% tpm2_policycommandcode(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_policycommandcode**(1) - Restrict TPM object authorization to specific TPM commands or operations.
+
+# SYNOPSIS
+
+**tpm2_policycommandcode** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policycommandcode**(1) Restricts TPM object authorization to specific TPM commands or operations. Useful when you want to allow only specific commands with the TPM object. Requires support for extended sessions with resource manager.
+
+# OPTIONS
+
+  * **-S**, **--session**=_SESSION\_FILE_:
+
+    A session file from **tpm2_startauthsession**(1)'s **-S** option.
+
+  * **-c**, **--command-code**=_TPM\_CC\_VALUE_:
```

Comment:
It would be nice to have a string CC support here, as well as a -l option to list all known CC and strings so people can build policy.

Reply:
agreed.

Reply:
Still open.

Reply:
Same as the other tool, command code should NOT be an option but an argument.